sdk/azidentity/azidentity.go

```diff
@@ -30,7 +64,12 @@ func (e *AuthenticationFailedError) Error() string {
 }
 
 func newAuthenticationFailedError(resp *azcore.Response) error {
-	var authFailed *AuthenticationFailedError
+	authFailed := &AuthenticationFailedError{}
+	if resp == nil {
```

Comment:
Remove this

Reply:
Fixed, added resp == nil code in the authenticate() methods in aad_identity_client